practice-one/pages/components.html

```diff
@@ -56,64 +57,68 @@ <h3 class="card-details">Mobile App, Website</h3>
     </div>
     <br>
 
-    <div class="card-item">
+    <div class="card-item blog-card">
       <figure class="card-header card-img">
-        <img src="../assets/images/blog-graphic-design.png" alt="Graphic Design"/>
+        <img src="../assets/images/blog-graphic-design.png" alt="Graphic design"/>
       </figure>
       <div class="card-body">
-        <h2 class="card-title">35 Stellar Graphic Design Blogs to Keep YouEducated and Inspired</h2>
+        <p class="card-title">35 Stellar Graphic Design Blogs to Keep YouEducated and Inspired</p>
         <div class="card-details">
-          <div class="details">
-            <img src="../assets/icons/icon-timer.svg" alt="icon time"/>
+          <div class="time-and-location">
+            <figure class="frame-img">
+              <img src="../assets/icons/icon-timer.svg" alt="icon time"/>
+            </figure>
             <span>January 25, 2021</span>
           </div>
 
-          <div class="details">
-            <img src="../assets/icons/icon-person.svg" alt="icon person"/>
+          <div class="time-and-location">
```

Comment:
Class time-and-location is not good for person and comment. If you want to use common class here, please choose another one